src/main/java/com/sahabuddin/contactmanager/controllers/UserController.java

```diff
@@ -35,8 +42,17 @@ public String userDashboard(Model model) {
     }
 
     @GetMapping(value = "/add-contact")
```

Comment:
Create a ContactController and then write all the contact related operation inside the controller.